desktop-app/app/components/Addressinput/index.js

```diff
@@ -178,6 +178,12 @@ class AddressBar extends React.Component<Props> {
             listItemUiClassName={cx(styles.searchBarSuggestionsListUl)}
             listItemsClassName={cx(styles.searchBarSuggestionsListItems)}
             activeClass={cx(styles.searchBarSuggestionsActiveListItems)}
+            pageMetaFavIconClassName={cx(styles.pageMetaFavIcon)}
```

Comment:
@JohnRawlins Can you please move these styles to file a `.css` file in `UrlSearchResults` component and directly access it instead of passing it from here?

Reply:
No problem.